lua/spectre_common.lua

```diff
@@ -23,6 +23,10 @@ local HEADERS = {
     ZIPKIN_ID = 'X-Zipkin-Id',
 }
 
+local POST_CACHEABLE_HEADERS = {
```

Comment:
This name is a bit confusing since I'd expect this to be the list of headers that we're caching.

Maybe call it supported post encoding? I don't think we need to make it more generic for now

Reply:
Fixed.